proto/cmp/services/info/v1alpha/entry_requirements.proto

```diff
@@ -0,0 +1,101 @@
+syntax = "proto3";
+
+package cmp.services.info.v1alpha;
+
+import "cmp/types/v1alpha/common.proto";
+import "cmp/types/v1alpha/country.proto";
+import "cmp/types/v1alpha/datetime_range.proto";
+import "cmp/types/v1alpha/filter.proto";
+import "cmp/types/v1alpha/language.proto";
+import "cmp/types/v1alpha/localized.proto";
+import "cmp/types/v1alpha/travel_type.proto";
+import "google/protobuf/timestamp.proto";
+
+// ![Diagram](https://storage.googleapis.com/docs-cmp-files/diagrams/proto/cmp/services/info/v1alpha/entry_requirements.proto.dot.xs.svg)
+// [Open Message Diagram](https://storage.googleapis.com/docs-cmp-files/diagrams/proto/cmp/services/info/v1alpha/entry_requirements.proto.dot.svg)
+service CountryEntryRequirementsService {
+  rpc CountryEntryRequirements(CountryEntryRequirementsRequest) returns (CountryEntryRequirementsResponse);
+}
+
+message CountryEntryRequirementsRequest {
+  // Message header
+  cmp.types.v1alpha.RequestHeader header = 1;
+
+  // Departure country
+  cmp.types.v1alpha.Country departure = 2;
+
+  // Destination country
+  cmp.types.v1alpha.Country destination = 3;
+
+  // Citizenship
+  cmp.types.v1alpha.Country citizenship = 4;
+
+  // Residence
+  cmp.types.v1alpha.Country residence = 5;
+
+  // Transit country
+  cmp.types.v1alpha.Country transit = 6;
+
+  // Travel type, ex: business or tourism
+  cmp.types.v1alpha.TravelType travel_type = 7;
+
+  // Date time range, start and end dates of visit, as timestamp.
+  cmp.types.v1alpha.DateTimeRange datetime_range = 8;
+
+  // Languages requested
+  repeated cmp.types.v1alpha.Language languages = 9;
+
+  bool exclude_categories = 10;
+
+  bool include_items = 11;
+
+  // Filters with type of `FilterType.FILTER_TYPE_A3M_ITEM_TYPE_KEYS`
+  repeated cmp.types.v1alpha.Filter filters = 12;
+}
+
+message CountryEntryRequirementsResponse {
+  // Message header
+  cmp.types.v1alpha.ResponseHeader header = 1;
+
+  // This must be a UUID according to RFC 4122
+  string response_id = 2;
+
+  // Contents
+  repeated CountryEntryRequirementCategory contents = 3;
+
+  // Items
+  repeated CountryEntryRequirementItem items = 4;
+}
+
+// Types
+
+message CountryEntryRequirementCategory {
+  // Category key. FIXME: Can this field be an enum?
+  string key = 1;
+
```

Comment:
maybe create a repeated message NameAndDescription with language, name and description fields?

Reply:
Updated